### ISSUE TITLE
elpy-refactor problem

### ISSUE THREAD

**Issue by dschaehi** [jorgenschaefer/elpy#1659]
# Summary
I get the following error message after calling `elpy-refactor`
```
elpy-refactor: Wrong type argument: listp, [((name . "refactor_organize_imports") (category . "Imports") (description . "Reorganize and clean up") (doc . "Clean up and organize imports.") (args . []) (available . t)) ((name . "refactor_module_to_package") (category . "Module") (description . "Convert the current module into a package") (doc . "Convert the current module into a package.") (args . []) (available . t)) ((name . "refactor_move_module") (category . "Module") (description . "Move the current module to a different package") (doc . "Move the current module.") (args . [["new_name" "directory" "Destination package: "]]) (available . t)) ((name . "refactor_rename_current_module") (category . "Module") (description . "Rename current module") (doc . "Rename the current module.") (args . [["new_name" "string" "Rename to: "]]) (available . t))]

```
# Steps to reproduce
M-x elpy-refactor

# My configuration
## OS
Linux Ubuntu 18.04

## Result of `(elpy-config)`
```
Emacs.............: 26.3
Elpy..............: 1.31.0
Virtualenv........: kgi (/home/jae/miniconda3/envs/kgi)
Interactive Python: ipython 7.8.0 (/home/jae/miniconda3/envs/kgi/bin/ipython)
RPC virtualenv....: elpy-rpc-venv (/home/jae/miniconda3/envs/elpy-rpc-venv)
 Python...........: python 3.7.4 (/home/jae/miniconda3/envs/elpy-rpc-venv/bin/python)
 Jedi.............: 0.15.1
 Rope.............: 0.14.0
 Autopep8.........: 1.4.4
 Yapf.............: 0.28.0
 Black............: 19.3b0
Syntax checker....: flake8 (/home/jae/miniconda3/envs/kgi/bin/flake8)
```

## Elpy configuration in my init.el
```
(use-package elpy
  :ensure t
  :defer t
  :bind
  (:map elpy-mode-map
        ("C-c f" . elpy-black-fix-code)
        ("M-RET" . elpy-shell-send-codecell-and-step))
  :hook (python-mode . elpy-mode)
  :config
  (when (require 'flycheck nil t)
    (setq elpy-modules (delq 'elpy-module-flymake elpy-modules))
    (add-hook 'elpy-mode-hook 'flycheck-mode))
  (elpy-enable)
  (cond ((eq system-type 'darwin)
         (setenv "WORKON_HOME" "/usr/local/miniconda3/envs"))
        ((eq system-type 'gnu/linux)
         (if (file-directory-p "~/miniconda3/")
             (setenv "WORKON_HOME" (concat (expand-file-name "~") "/miniconda3/envs"))
             (setenv "WORKON_HOME" (concat (expand-file-name "~") "/.conda/envs")))))
  (defalias 'workon 'pyvenv-workon)
  (call-interactively 'workon)
  ;; (setq python-shell-interpreter "python"
  ;;     python-shell-interpreter-args "-i")
  (setq python-shell-interpreter "ipython"
        python-shell-interpreter-args "-i --simple-prompt")
  (setq elpy-rpc-timeout 10)
  (setq elpy-shell-echo-input t)
  (setq elpy-shell-echo-input-lines-head 2)
  (setq elpy-shell-echo-input-lines-tail 2)
  (add-hook 'elpy-mode-hook 'hs-minor-mode))
```

**Comment by dschaehi** [comment]
elpy-refactor works again with the recent commit (elpy-20190910.2224) in MELPA.

**Comment by galaunay** [comment]
Thank you for letting us now.

This should indeed have been fixed by ec2f74a.
